.solhint.json

```diff
@@ -4,6 +4,17 @@
   "rules": {
     "quotes": ["error", "single"],
     "compiler-version": ["error", "^0.8.0"],
-    "func-visibility": ["warn", { "ignoreConstructors": true }]
+    "func-visibility": ["warn", { "ignoreConstructors": true }],
+    "no-inline-assembly": "off",
+    "no-empty-blocks": "off",
+    "avoid-low-level-calls": "off",
+    "not-rely-on-time": "off",
+    "no-complex-fallback": "off",
+    "const-name-snakecase": "off",
+    "var-name-mixedcase": "off",
+    "func-name-mixedcase": "off",
+    "state-visibility": "off",
+    "reason-string": "off",
+    "no-global-import": "off"
```

Comment:
these are good practices that we can adhere too, don't need to ignore, and we should resolve instead